WcaOnRails/app/models/user.rb

```diff
@@ -508,6 +508,11 @@ def editable_fields_of_user(user)
       fields << :senior_delegate_id
       fields << :region
     end
+    if user.any_kind_of_delegate? && (user == self || user.senior_delegate == self || admin? || board_member)
+      fields << :location_description
+      fields << :phone_number
+      fields << :notes
```

Comment:
Nope, I probably wasn't aware that was an option when I started. I would worry about 1 line growing really long this way, however. Can it be split up into multiple lines?